src/opendkim.h

```diff
@@ -1,6 +1,10 @@
 #ifndef LIBOPENDKIM_H
 #define LIBOPENDKIM_H
 
+#if __GNUC__ > 4
+#define _Bool bool
```

Comment:
Modern C has `<stdbool.h>` that does this for you--not sure if that's useful in this context.

Reply:
this is what failed, the new header for gcc >4 doesn't work like the old one. :/